plugins/chat/assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -31,6 +31,7 @@ const THROTTLE_MS = 150;
 export default Component.extend(TextareaTextManipulation, {
   chatChannel: null,
   chat: service(),
+  chatChannelComposer: service(),
```

Comment:
I don't think this is quite right, you are passing in `composerService` from `chat-live-pane` (which is correct, since `chat-live-pane` knows what correct composer service to use) and then not using it. Will fix this up

Reply:
Ah sorry I saw you did it with `ChatMessage`, but I think the point still stands